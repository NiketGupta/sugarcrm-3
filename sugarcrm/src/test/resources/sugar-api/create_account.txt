method=set_entry
&input_type=json
&response_type=json
&rest_data={
   "session":"li1o8lsb8o37gqres0bf7l88u3",
   "module_name":"Accounts",
   "name_value_list":[
      {
         "name":"name",
         "value":"ACME Inc."
      }
   ]
}
