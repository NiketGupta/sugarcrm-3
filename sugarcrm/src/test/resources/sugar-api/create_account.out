{"id":"9847d9b2-4009-aa6a-821e-52b33dcfb0f2","entry_list":{"name":{"name":"name","value":"ACME Inc."}}}